build.gradle

```diff
@@ -29,7 +29,7 @@ dependencyUpdates.resolutionStrategy = {
     }
 }
 
-def springBootVersion = '2.2.5.RELEASE'
+def springBootVersion = '3.0.3'
```

Comment:
I am bit curious to know this, is this 3.0.3 version released? 
https://mvnrepository.com/artifact/org.springframework.boot/spring-boot

Reply:
was typo. corrected it.